src/components/panel/panelSections/panelContent.component.tsx

```diff
@@ -105,76 +114,97 @@ export const PanelContent = ({
     onReorder(updatedObjects);
   };
 
+  if (draggedObject)
```

Comment:
We can add a unit test for this because you're passing `draggedObject` in as a prop